docs/CALIBRATION_GUIDE.md

```diff
@@ -0,0 +1,73 @@
+# Camera Calibration 
+
+The purpose of camera calibration is to gain a quantitative understanding of how the cameras relate to each other in space and how they differ from each other due to their internal mechanisms. 
+To model the spatial relationships of cameras and their internal specifications, we rely on 2 sets of matrices for each camera, the extrinsics and intrinsics matrices respectively. Once we have these sets of matrices, we can then triangulate 3D poses from our tracked 2D poses. 
+
+# Practical Considerations
+
+## Calibration Board 
+
+A calibration board is a rectangular surface with a recognizable pattern that allows for quick detection via modern computer vision software. The most commonly used types of boards are checkerboards, aruco boards (which are rectangular grids with unique markers), and charuco boards (which are a combination of the two). 
+
+**INSERT IMAGE HERE**
+
+Calibration boards are commonly made by pasting a printed board design onto a rigid surface, however, they can also be produced via laser cutting into acryllic or other methods. Regardless of the method, it is paramount that the board design is clearly visible to the cameras. 
+
+One can create a charuco board pattern via two different ways using sleap-anipose.
+
+1. Through the CLI
+
+```
+slap-write_board --board_name board --img_format jpg --board_X 8 --board_Y 11 --square_length 24.0 --marker_length --18.75
+```
+
+2. Through the API
+
+```
+python 
```

Comment:
I think you meant for `python` to be on the same line as the backticks to enable syntax highlighting?

Reply:
resolved